### ISSUE TITLE
"Cancel quickMove" feature missing / removed from 6.10?

### ISSUE THREAD

**Issue by burval** [RealRaven2000/QuickFolders#558]
The option to "cancel quickMove" appears to be missing as of version 6.10...

![Image](https://github.com/user-attachments/assets/ef2b7f68-d041-4e4e-89a3-ad886df2db5f)

Previously the above menu included a "cancel quickMove" node:

![Image](https://github.com/user-attachments/assets/0e1155e4-2ab0-4511-89a3-2c531076f4e0)

Hoping the "cancel quickMove" option will be restored to this menu in a future release.

**Comment by RealRaven2000** [comment]
I will look into it, but if  you have focused the search box (so you can type in it) you can cancel it by pushing the [ESC] key on the keyboard. This option can be enabled in 

Settings >> Licenses >> Advanced Search Settings

![Image](https://github.com/user-attachments/assets/c6f0361e-f7c1-4cca-b92d-2828f0c77e89)

in quickMove Advanced Settings there is a setting
"Escape Key always clears messages list."

could this be related?

**Comment by RealRaven2000** [comment]
I found the cause of the problem I was using collapsed="true" in the previous version but I also now added hidden="true" to avoid screen readers going there accidentally.

fixed version:

[QuickFolders-mx-6.10.1pre44.zip](https://github.com/user-attachments/files/19352499/QuickFolders-mx-6.10.1pre44.zip)

----

To try out this version, download the zip file and then drag it into Thunderbird Add-ons Manager (without extracting)
for users of screenreader you can also install this via the tools button after the "Find more addons" search text. activate the button with Enter then chose the 3rd menu entry "install addon from file...". Then select the zip file to install.

Please leave the bug open even if this fixes it. I am always closing bugs on official release date.

**Comment by burval** [comment]
Impressive turnaround time @RealRaven2000 thanks for the fix.  Working great on my instance.  I already hold a Pro license and renew annually without hesitation but would like to make an additional donation, please advise on how I can do so.

**Comment by QuinnyNZ** [comment]
Came to post a bug because, OMG I was banging my head on this for the last hour. Thank you so much for the quick fix. I have massive folders and without cancel Im in a mess.

**Comment by RealRaven2000** [comment]
> Came to post a bug because, OMG I was banging my head on this for the last hour. Thank you so much for the quick fix. I have massive folders and without cancel Im in a mess.

no worries, I might publish the fixed version already the end of the week. Working hard on accessibility so I can better test with my blind users. Hence we are already into revision 44 for the maintenance version 6.10.1. See also #552 for all the current activity that causes this temporary disruption.

**Comment by QuinnyNZ** [comment]
> no worries, I might publish the fixed version already the end of the week. Working hard on accessibility so I can better test with my blind users. Hence we are already into revision 44 for the maintenance version 6.10.1. See also [#552](https://github.com/RealRaven2000/QuickFolders/issues/552) for all the current activity that causes this temporary disruption.

Oh, that's awesome. My Sister is Blind with 100% loss, so anyone working on that is appreciated by me :) A huge HUGS to you.

**Comment by RealRaven2000** [comment]
Fixed  in 6.10.1. - Published 20/03/2025
